config/charts/inferencepool/templates/gke.yaml

```
@@ -1,4 +1,4 @@
-{{- if eq .Values.provider.name "gke" }}
+{{- if or (eq .Values.provider.name "gke") (eq .Values.provider.name "GKE") }}
```

I think this may end up being confusing. 
being case insensitive means “Gke” is also valid value. 

I recommend converting both strings to lower case and compare, and in that case we can say we’re case insensitive. 
```suggestion
{{- if eq (lower .Values.provider.name) "gke" }}
```